rocAL/rocAL/source/parameters/parameter_rali_crop.cpp

```diff
@@ -58,8 +58,19 @@ void RocalCropParam::fill_crop_dims()
             // Evaluating user given crop
             (crop_w > in_width[img_idx]) ? (cropw_arr_val[img_idx] = in_width[img_idx]) : (cropw_arr_val[img_idx] = crop_w);
             (crop_h > in_height[img_idx]) ? (croph_arr_val[img_idx] = in_height[img_idx]) : (croph_arr_val[img_idx] = crop_h);
```

Comment:
The above 2 lines can be changed this way

cropw_arr_val[img_idx] = (crop_w > in_width[img_idx]) ? in_width[img_idx] : crop_w;

croph_arr_val[img_idx] = (crop_h > in_height[img_idx]) ? in_height[img_idx] : crop_h;